tests/forte/data/entry_type_generator_test.py

```diff
@@ -1,10 +1,17 @@
 import unittest
 from unittest.mock import patch
 from forte.data.entry_type_generator import EntryTypeGenerator
+from ft.onto.base_ontology import Token
+from ft.onto.wikipedia import WikiPage
```

Comment:
Before a full review from qinxin, I would like to ask a question.

I heard that we have to import a type here to make it appear in the generated list. Can we get around this? For our use case, it is impossible to import them beforehand.

Reply:
In the comment section of the highest voted answer here: https://stackoverflow.com/questions/3862310/how-to-find-all-the-subclasses-of-a-class-given-its-name, they have mentioned a method.

Reply:
let us know if you find this impossible though, we will consider alternate approaches.

Reply:
OK， I will try to find it is possible to get around this.

Reply:
> In the comment section of the highest voted answer here: https://stackoverflow.com/questions/3862310/how-to-find-all-the-subclasses-of-a-class-given-its-name, they have mentioned a method.

I also used this method to generate the ontology tree.  But please note if the class definition of a subclass hasn't been executed yet - for example, if the subclass's module hasn't been imported yet - then that subclass doesn't exist yet, and __subclasses__ won't find it.

Reply:
OK. I have attached two screenshots of the test result for easy viewing.  

![image](https://user-images.githubusercontent.com/11881183/154635432-32b87c7a-4d6f-4765-a722-821c3c1fb767.png)
![image](https://user-images.githubusercontent.com/11881183/154635506-5e16e3e9-65cc-463c-b63a-a27d30300daf.png)

Reply:
> 

Also,  we do not need to import all the types, Like the below images, we just import Token, all classes that belong to base_ontology are appear in the generated list. 
![image](https://user-images.githubusercontent.com/11881183/154641694-60702ddf-885e-4480-9168-610d26ec5d97.png)

Reply:
So based on xuezhi's investigation (thanks on that!). We will have to import at least something that contains the ontology class.

However, this will make this function not match our expectations.  Since we have no way to ensure we import every possible ontology class before running this function. So we may consider using an alternative approach to solve this problem, which does not require we import all classes upfront.

Thanks to Xuezhi on this, if we decided to use the alternate approach, we can consider this issue solved as not-fixable.

We will still work on the attribute structure given a entry name, but that will be a slightly different issue.

Reply:
Thanks, @hunterhector， Once the team has the final decision, please let me know, as #583 and #584 meet the same problems.